test/constraint.jl

```diff
@@ -1,4 +1,8 @@
-@testset "Constraints" begin
+# The parameter names correspond to the JuMP types so that the test can serve as examples that can be copy pasted
+function constraints_test(Model::Type{<:JuMP.AbstractModel}, VariableRef::Type{<:JuMP.AbstractVariableRef})
+    AffExpr = JuMP.GenericAffExpr{Float64, VariableRef}
```

Comment:
Just use `JuMP.AffExpr` and `JuMP.QuadExpr`?

Reply:
The names of the function correspond to the name of the JuMP types so that examples can be copy pasted but they may not be. So `VariableRef` may not be equal to `JuMP.VariableRef`.

Reply:
That's confusing, use variable names that don't conflict with existing type names, e.g., `ModelType`, `VariableRefType`, `AffExprType`.